proto/buf/alpha/registry/v1alpha1/repository_commit.proto

```diff
@@ -47,6 +47,8 @@ message RepositoryCommit {
   // https://spdx.org/licenses, and will be not set if the license file is not
   // presented or cannot be classified into a known license.
   string spdx_license_id = 10;
+  // The manifest digest of the commit. Populated via Tamper Proofing.
+  string manifest_digest = 11;
```

Comment:
About the deprecation, I prefer to do that at the same time we mark as deprecated the old Push/Download rpcs. For now since everything is backwards compatible, no need for that additional warning.

For the `module_resolver` and other CLI code changes let's do in an upcoming PR? I'd like this to be schema change only.